app/services/organizations/create_service.rb

```diff
@@ -51,6 +56,16 @@ def create_organization(name, slug, email)
     )
   end
 
+  def create_location(country, city_town, province_state)
+    ActsAsTenant.with_tenant(@organization) do
```

Comment:
RE `ActsAsTenant.with_tenant(@organization)` I believe we no longer need to scope this resource to a tenant (organization) now that it's polymorphic as it could belong to multiple resources

Reply:
@kasugaijin Gotcha! Just finished merging with the current changes to update the branch and implemented the change. ActAsTenant is no longer scoping location to the organization with my removing it, and I've added in the associating key locatable to the method pointing it to organization as its parent. Tests are passing on my end and I've pushed up my changes. Is there anywhere else that might need refactoring now that location is polymorphic or is this set to be merged?